src/core/execute.c

```diff
@@ -5659,6 +5665,19 @@ void exec_context_dump(const ExecContext *c, FILE* f, const char *prefix) {
                                 o->options);
                 fprintf(f, "\n");
         }
+
+        for (size_t i = 0; i < c->n_extension_images; i++) {
+                MountOptions *o;
+
+                fprintf(f, "%sExtensionImages: %s%s", prefix,
+                        c->extension_images[i].ignore_enoent ? "-": "",
+                        c->extension_images[i].source);
+                LIST_FOREACH(mount_options, o, c->extension_images[i].mount_options)
+                        fprintf(f, ":%s:%s",
+                                partition_designator_to_string(o->partition_designator),
+                                o->options);
```

Comment:
isn't `strempty()` necessary here?

Reply:
whoops, good catch - fixed in a separate commit for MountImages too, since that was borken as well

Reply:
actually split the MountImages one out since it's unrelated: https://github.com/systemd/systemd/pull/18728